### DISCUSSION TITLE
Yelp Open Dataset

### DISCUSSION THREAD

**Discussion by trafficpest** [teambtcmap/btcmap-data#641] [General]
Just pulled this from Yelp's Open data set
found 470 businesses just in that too bad I don't work at yelp

[yelp_bitcoin.txt](https://github.com/teambtcmap/btcmap-data/files/9804509/yelp_bitcoin.txt)

**Comment by trafficpest** [comment]
I just dont understand why would you collect that data on every business but not have an option to search for it on your website?

**Reply by secondl1ght** [reply]
Which data are you referring to? We will be adding search features in the future.

**Reply by trafficpest** [reply]
Yelp, they collect if businesses accept bitcoin on every location but they dont have a search option or filter on the page. I pulled that data from here https://www.yelp.com/dataset they release some of there data every few years in json format for academic purposes.

**Reply by secondl1ght** [reply]
Ah ok gotcha, thanks for looking into that data. Is any of it recent like within the last year? It looks like there is no date in the metadata and even no bitcoin tags that I can see. Are there more fields that were not included in the export?

**Reply by trafficpest** [reply]
every yelp location has a "BusinessAcceptsBitcoin": boolean argument. All the ones in the text are 'True' its right after parking

**Comment by trafficpest** [comment]
looks like this one came out 4/20/2022  from 150,346 businesses (randomly selected?) translating to roughly a 0.31% merchant adoption. Lots of room to grow

**Reply by secondl1ght** [reply]
Interesting that they do this, maybe other platforms are also starting to include bitcoin information in their datasets as well. Yep we have a long way to go before it becomes the norm!
